modules/c/nitf/source/BandSource.c

```diff
@@ -57,9 +57,10 @@ NITFPRIV(NITF_BOOL) MemorySource_contigRead(
         nitf_Off size,
         nitf_Error* error)
 {
+    (void) e; /* Suppresses a warning. Param seems to exist for consistency purposes. Not used. */
```

Comment:
Can you make sure this compiles?  Lines like this won't compile right (`e` vs. `error`)?